cmd/main.go

```diff
@@ -70,7 +73,15 @@ func main() {
 		Name:      specs.SchemasConfigMapName,
 		Namespace: specs.SchemasConfigMapNamespace,
 	}
-	router := web.NewRouter(idpConfig, schemasConfig, hAdminClient, kAdminClient, tracer, monitor, logger)
+
+	rulesConfig := &rules.Config{
```

Comment:
as much as this is cumbersome, standardization and conventions are quite important

i would stick with a func called `NewConfig` even if we are not really initializing anything atm
see https://github.com/canonical/identity-platform-admin-ui/pull/157/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R51

Reply:
done